h5py/_hl/files.py

```diff
@@ -16,6 +16,19 @@
 import sys
 import os
 
+try:
```

Comment:
is there a way to hide this somewhere? perhaps in `compat.py` itself? also it's duplicated in `group.py`.